src/HtmlGenerator/Smart.Design.HtmlGenerator/Program.cs

```diff
@@ -0,0 +1,42 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+using Smart.Design.HtmlGenerator.Managers;
+using Smart.Design.Razor.Extensions;
+using LogLevel = Microsoft.Extensions.Logging.LogLevel;
+namespace Smart.Design.HtmlGenerator;
+
+/// <summary>
+///CLI for generating HTML code from custom tag helpers
+/// </summary>
+public class Program
```

Comment:
Seeing as we are using .NET 6, you can get rid of a lot of boiler code:

- Implicit usings: https://www.hanselman.com/blog/implicit-usings-in-net-6 
- Top level statements: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements